contrib/confluentinc/confluent-kafka-go/kafka/option.go

```diff
@@ -8,24 +8,31 @@ package kafka
 import (
 	"context"
 	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
 )
 
 type config struct {
-	ctx           context.Context
-	serviceName   string
-	analyticsRate float64
+	ctx                 context.Context
+	consumerServiceName string
+	producerServiceName string
+	analyticsRate       float64
 }
 
 // An Option customizes the config.
 type Option func(cfg *config)
 
 func newConfig(opts ...Option) *config {
 	cfg := &config{
-		serviceName: "kafka",
-		ctx:         context.Background(),
+		ctx: context.Background(),
 		// analyticsRate: globalconfig.AnalyticsRate(),
 		analyticsRate: math.NaN(),
 	}
+	cfg.consumerServiceName = "kafka"
```

Comment:
Can we just declare the defaults above in the inline `cfg` structure declaration for both service names and then use the same as suggested above here?

```go
if svc := globalconfig.ServiceName(); svc != "" {
    cfg.producerServiceName = svc
}
```